tests/urlpatterns_reverse/tests.py

```diff
@@ -700,7 +700,17 @@ def test_urlconf_overridden(self):
         ]
     )
     def test_urlconf_overridden_with_null(self):
-        self.assertRaises(ImproperlyConfigured, self.client.get, '/test/me/')
+        """
+        Test that overriding request.urlconf with None will fall back to the
```

Comment:
For new code, I try to avoid the "Test that" prefix as the purpose of all tests is to test things. Instead just state the expected behavior, "Setting request.urlconf.=None will cause the the default URLconf to be used."